packages/image_picker/image_picker/example/ios/RunnerUITests/ImagePickerFromGalleryUITests.m

```diff
@@ -165,7 +165,7 @@ - (void)launchPickerAndPick {
   // Find an image and tap on it. (IOS 14 UI, images are showing directly)
   XCUIElement* aImage;
   if (@available(iOS 14, *)) {
-    aImage = self.app.scrollViews.firstMatch.images.firstMatch;
+    aImage = [self.app.scrollViews.firstMatch.images elementBoundByIndex:1];
```

Comment:
This is because there is a  [known issue](https://www.google.com/search?q=63426347+apple&ei=XdSLYJqTIP-O9u8P54eH8AU&oq=63426347+apple&gs_lcp=Cgdnd3Mtd2l6EAxQAFgAYIc0aABwAngAgAE5iAE5kgEBMZgBAKoBB2d3cy13aXrAAQE&sclient=gws-wiz&ved=0ahUKEwjatcCJ2qXwAhV_h_0HHefDAV4Q4dUDCA0) to pick HEIC images with PHPicker implementation. But only for simulators. So we have two options here. Either test with real device or test with non-HEIC images until Apple solves the issue.